tools/reduce/reduce.cpp

```diff
@@ -110,6 +110,22 @@ Options (in lexicographical order):
                reducer will take before giving up.
   --version
                Display reducer version information.
+
+Supported validator options:
+  --relax-logical-pointer          Allow allocating an object of a pointer type and returning
+                                   a pointer value from a function in logical addressing mode
+  --relax-block-layout             Enable VK_KHR_relaxed_block_layout when checking standard
+                                   uniform, storage buffer, and push constant layouts.
+                                   This is the default when targeting Vulkan 1.1 or later.
+  --scalar-block-layout            Enable VK_EXT_scalar_block_layout when checking standard
+                                   uniform, storage buffer, and push constant layouts.  Scalar layout
+                                   rules are more permissive than relaxed block layout so in effect
+                                   this will override the --relax-block-layout option.
+  --skip-block-layout              Skip checking standard uniform/storage buffer layout.
+                                   Overrides any --relax-block-layout or --scalar-block-layout option.
+  --relax-struct-store             Allow store from one struct type to a
+                                   different type with compatible layout and
+                                   members.
```

Comment:
^ How painful would it be to factor out the text for these help options?  Their descriptions could well change in the validator, and it would be a shame for them to get out of sync.  I appreciate that with the validator being a separate project this might not be easy.